guides-and-standards/standards--source-control.md

```diff
@@ -0,0 +1,153 @@
+
+## Data management
+
+- For publicly available data, scripts used to download and process these data should be preserved, as should the versions of items used in processing (e.g. probe to gene mappings). This code should be version controlled. Where possible, intermediate files of reasonable size can be stored to facilitate re-use, but the process to regenerate these files from publicly available data should be preserved.
+- Files under 100MB in size can be stored in the repository.
+- Private files over 100MB should be stored in a cloud storage service (e.g. an S3 bucket or Google Drive) and instructions to access the files should be included in the README.
+- Public files should be stored in a public archive (e.g. the Sequence Read Archive), in a Zenodo repository, or in a publicly-accessible S3 bucket. Reach out to the software team for help with this.
+- When we generate data, they should be stored in a location where they are replicated and uploaded to the relevant database as soon as possible (e.g. raw sequencing data uploaded to the Sequence Read Archive). If possible, the code should be updated to match the above. For more on this, please see the relevant publishing documentation [here](https://www.notion.so/arcadiascience/Sharing-Data-eca60da150f04a7a905a8540871f0364).
+
+## Guidelines for collaborating on codebases with collaborators who are not Arcadians
+
+- Before beginning a collaboration, please consult with your manager and our legal counsel to make sure there are not licensing and legal problems with your collaboration.
+- If you’re working on an Arcadia Science repository with outside collaborators, the repository is most likely already public. If so, that’s great! If the repository is private, you can include the Github account to the repository as an “outside collaborator”.
+- Either way, the rules in this document as it pertains to code review and working in branches apply to outside collaborators as well.
+- If you plan to actively work on a repository with outside collaborators, it’s recommended to add `CONTRIBUTING.MD` and `CODE_OF_CONDUCT.md` files to the root of the repository to set expectations ahead of time. Default templates for these will be available soon.
+
+## Common pitfalls and tricks to overcome them
+
+Git is notoriously difficult to use. It’s easy to get stuck in a quagmire of git branches and have no idea how to work your way out. If this happens to you, we’ve all been there, don’t panic! ChatGPT and/or Google are a good first option to see if you can identify a solution. If not, post in the #software-questions Slack channel to get help from fellow Arcadian. See below for some common pitfalls and some strategies to avoid or overcome them.
```

Comment:
```suggestion
Git is notoriously difficult to use. It’s easy to get stuck in a quagmire of git branches and have no idea how to work your way out. If this happens to you, we’ve all been there, don’t panic! ChatGPT and/or Google are a good first option to see if you can identify a solution. If not, post in the #software-questions Slack channel to get help from a fellow Arcadian. See below for some common pitfalls and some strategies to avoid or overcome them.
```